code/game/objects/structures/statues.dm

```diff
@@ -276,3 +282,4 @@
 	max_integrity = 400
 	icon_state = "dimas"
 	oreAmount = 10 //dimas b dense
+	impressiveness = -50 //ugly ass
```

Comment:
I too hate this reference, but don't give it a memely low number as it's attainable in game, and it couldn't possible be *that* ugly

Reply:
Yeah... I was probably *too* spiteful here

Reply:
lowered the value to -5 in f661421